ffs/api/api.go

```diff
@@ -96,9 +106,43 @@ func (i *API) ID() ffs.APIID {
 	return i.cfg.ID
 }
 
-// WalletAddr returns the Lotus wallet address.
-func (i *API) WalletAddr() string {
-	return i.cfg.WalletAddr
+// WalletAddr returns the wallet addresses.
+func (i *API) WalletAddrs() map[string]AddrInfo {
+	return i.cfg.Addrs
+}
+
+func (i *API) GetDefaultCfg() ffs.DefaultCidConfig {
+	return i.cfg.DefaultCidConfig
+}
+
+// NewAddr creates a new address managed by the FFS instance
+func (i *API) NewAddr(name string, options ...NewAddressOption) (string, error) {
+	i.lock.Lock()
+	defer i.lock.Unlock()
+
+	conf := &NewAddressConfig{
+		makeDefault: false,
+		addressType: defaultAddressType,
+	}
+	for _, option := range options {
+		option(conf)
+	}
+
+	addr, err := i.wm.NewAddress(context.Background(), conf.addressType)
```

Comment:
I think we should ask for a context in the method, since this is making a network call.